packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -188,7 +188,6 @@ export function LinkUI( props ) {
 			<LinkControl
 				hasTextControl
 				hasRichPreviews
-				className={ props.className }
```

Comment:
I don't think we should do this. Link UI may need to pass a `className` and we're removing that ability.

Reply:
The `LinkControl` component [does not receive `className`](https://github.com/WordPress/gutenberg/blob/2f0e60cb47a8eda7c927d634e87c521888739bf6/packages/block-editor/src/components/link-control/index.js#L117-L135), should it still be left?